src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,13 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @NotNull
+    @Size(min = 1)
     private int capacity;
```

Comment:
can int be null?
and can @Size be applied to int?

Reply:
)) Ok

Reply:
can @Size be applied to int?

Reply:
Yes, need @Min and @Max supports primitives and their wrappers